sysutils/john/Portfile

```diff
@@ -167,9 +167,12 @@ if {${subport} in [list "john-jumbo" "john-jumbo-devel"]} {
     if {${subport} eq "john-jumbo-devel"} {
         depends_run-append      port:py${python3_version}-asn1 \
                                 port:py${python3_version}-cbor2 \
+                                port:py${python3_version}-hanko \
                                 port:py${python3_version}-lxml \
-                                port:py${python3_version}-pycryptodome \
-                                port:py${python3_version}-scapy
+                                port:py${python3_version}-pycryptodome
+
+        # allow installing both: py-scrapy and py-scrapy-devel
+        depends_run-append      path:${frameworks_dir}/Python.framework/Versions/python${python3_branch}/site-packages/scrapy/__init__.py:py${python3_version}-scapy
```

Comment:
@reneeotten can it be cleaner?

Reply:
probably not unless you use the "python PG"

Reply:
Not sure that it help, because I use two different version of python at the same time here.

Reply:
that should help. The only thing you need to check for is, for example,whether either py311-scrapy or py311-scrapy-devel is installed. (Of course you shouldn't hardcode the "3.11" in the check, that should rely on a version of ${python3_version} that contains a ".")

Reply:
I like that, thank!

Reply:
I'll care of remaining  scapy dependencies as part of https://github.com/macports/macports-ports/pull/21853